docker/Dockerfile-quickstart

```diff
@@ -0,0 +1,15 @@
+# This Dockerfile governs the uPortal Quickstart Docker image.  The portal must be fully assembled
+# (based on tomcatInstall + tomcatDeploy) before building the Docker image.
+
+# This image requires Gradle;  it is not as slim as the web server-only image
+FROM apereo/uportal-cli
+
+# The behavior of this image is based on a shell script
+COPY etc/docker/quickstart.sh quickstart.sh
+RUN chmod +x quickstart.sh
+
+# TCP traffic from this image is available on port 8080 (in case the Tomcat instance is started)
+EXPOSE 8080
+
+# Containers based on this image are expected to invoke the Gradle-based CLI
+ENTRYPOINT ["./quickstart.sh"]
```

Comment:
would
```dockerfile
CMD gradle --no-daemon hsqlStart dataInit && .gradle/tomcat/bin/catalina.sh run
```
work?

Reply:
My sense is that `ENTRYPOINT` is more idiomatic in this case than `CMD`.  I guess I agree with the things I'm reading, like...

> In contrast, ENTRYPOINT should be used in scenarios where you want the container to behave exclusively as if it were the executable it's wrapping. That is, when you don't want or expect the user to override the executable you've specified.

The `uportal-cli` image has an `ENTRYPOINT` (`gradle`) and a `CMD`, with the specific expectation that users will likely want to override the `CMD` (easy) but not the `ENTRYPOINT` (harder).

I expect the `uportal-quickstart` image to do one thing:  run the uPortal Quickstart experience.

I like organizing the startup commands into a `.sh` file b/c it might get more verbose.  Down the road, we may have 3-4 commands, and a fair number of comments in that file.  That would be clumsy(er) written directly into the Dockerfile.